src/main/java/com/celements/course/service/ICourseServiceRole.java

```diff
@@ -50,7 +51,11 @@ public long getRegistrationCount(@NotNull DocumentReference courseDocRef)
 
   @NotNull
   public long getRegistrationCount(@NotNull DocumentReference courseDocRef,
-      @Nullable RegistrationState state) throws LuceneSearchException;
+      @Nullable ParticipantStatus state) throws LuceneSearchException;
+
+  @NotNull
+  public long getRegistrationCount(DocumentReference courseDocRef,
+      List<ParticipantStatus> ignorList) throws LuceneSearchException;
```

Comment:
naming (everywhere in your changes, not just this line):
- "ignore" with e
- the name should state what is ignored, e.g. ignoreStates